.gitignore

```
@@ -1,5 +1,5 @@
 # Dependencies
-/node_modules
+**/node_modules
```

```suggestion
node_modules
```

Not including the leading slash should ignore any `node_modules` in the current directory and any subdirectories.

---

Lol. And i already wondered. Totally missed the slash 😅